plugins/woocommerce/src/Internal/Admin/FeaturePlugin.php

```diff
@@ -68,6 +75,11 @@ public function init() {
 			return;
 		}
 
+		if ( $this->initialized ) {
```

Comment:
Curious: if this is a temporary solution for the CI to run, and are we going to have a more permanent solution later?

Reply:
The class is intended to be initialized only once anyway, so in principle it doesn't hurt to leave the check in place; should it cause isses somehow, we can think of an alternative solution.